src/views/product-tutorials-view/helpers/generate-headings.ts

```diff
@@ -0,0 +1,18 @@
+import slugify from 'slugify'
+
+//@TODO type blocks
```

Comment:
Waiting to type blocks until we copy over the component files, deferring that in case the interfaces change at all.